core/uvisor.h

```diff
@@ -68,7 +68,7 @@ typedef void (*UnprivilegedBoxEntry)(void);
 /* IOT-OS specific includes */
 #include <linker.h>
 
-/* system specific declatiions and IRQ vector table */
-#include <system.h>
+/* System IRQ vector table and default ISRs */
+#include "system.h"
```

Comment:
This seems like a purely cosmetic change. Probably it shouldn't be part of this commit.